### DISCUSSION TITLE
FTP delete doesn't work

### DISCUSSION THREAD

**Discussion by danilklimenko** [mikekazakov/nimble-commander#341] [Development]
I've tried it with Proftpd, Pure-FTPd instances.
Files are created/uploaded/overwritten correctly but when I try to delete them I get an error: "The operation couldn’t be completed. Operation not permitted".
I've checked FTP logs files - it's empty, so Nimble Commander Pro doesn't even try to execute deletion.
Am I doing something wrong or it's just not implemented yet?

Thanks in advance!

<img width="457" alt="Monosnap Monosnap 2024-07-19 18-14-47" src="https://github.com/user-attachments/assets/44012f0b-c1c3-4f9f-8fc9-9160aca9da1f">

**Comment by mikekazakov** [comment]
That's implemented for sure.
Can you pick the latest nightly build and check the logs with it?
Make sure to set logging level to vfs->trace.
Here's an example of how it looks when successfully removing the file and refreshing the listing:
<img width="919" alt="Screenshot 2024-07-19 at 19 51 57" src="https://github.com/user-attachments/assets/8dff3322-c3c5-4623-aafe-bfc4d73bff91">

**Comment by danilklimenko** [comment]
Thanks, it works!
I've updated NC 1.5 -> 1.6 and found out that it's free and open-source now =)
